cmd/root.go

```diff
@@ -76,6 +75,72 @@ func initRoot(cmd *cobra.Command, args []string) {
 	}
 }
 
+type Credentials struct {
+	Credentials struct {
+		AppTerraformIo struct {
+			Token string `json:"token"`
+		} `json:"app.terraform.io"`
+	} `json:"credentials"`
+}
+
+func getToken() {
+	credentials, err := readTerraformCredentials()
+	if err != nil {
+		errLog.Fatalln("failed to get Terraform credentials:", err)
+	}
+
+	if credentials != nil {
+		token := credentials.Credentials.AppTerraformIo.Token
+		if token != "" {
+			lib.SetToken(token)
+			return
+		}
+	}
+
+	// fall back to using ATLAS_TOKEN environment variable
+	atlasToken := os.Getenv("ATLAS_TOKEN")
+	if atlasToken != "" {
+		lib.SetToken(atlasToken)
+		return
+	}
+
+	errLog.Fatalln("no credentials found, use 'terraform login' to create a token")
+}
+
+func readTerraformCredentials() (*Credentials, error) {
+	userConfigDir := os.UserHomeDir
+	if runtime.GOOS == "windows" {
+		userConfigDir = os.UserConfigDir
+	}
+
+	var err error
+	configDir, err := userConfigDir()
+	if err != nil {
+		return nil, fmt.Errorf("unable to get the home directory: %v", err)
+	}
+
+	credentialsPath := filepath.Join(configDir, ".terraform.d", "ceredentials.tfrc.json")
```

Comment:
This looks like a typo
```suggestion
	credentialsPath := filepath.Join(configDir, ".terraform.d", "credentials.tfrc.json")
```

Reply:
Good catch. It was an intentional change I forgot to reverse.